src/user/accounts-iframe.cjsx

```diff
@@ -68,6 +68,23 @@ AccountsIframe = React.createClass
   componentWillMount: ->
     window.addEventListener('message', @parseAndDispatchMessage)
 
+  safariWarning: ->
+    browser = window.navigator.userAgent
+    return unless @props.type is 'login' and _.contains(browser, 'Safari') and not _.contains(browser, 'Chrome')
+    # strip off the pathname
+    url = User.endpoints.accounts_iframe.replace(/openstax.org\/.*/, 'openstax.org/')
```

Comment:
Could we use an `<a>` element to pull out the hostname? or isn't it already defined in the CC widget config?